app/controllers/cart_controller.rb

```diff
@@ -20,6 +22,10 @@ def remove_item
     redirect_to '/cart'
   end
 
+  def deny_admin
```

Comment:
Should this be a private method?

Reply:
We should decide a convention in our next group meeting

Reply:
I think its fine as is for now